src/App.js

```diff
@@ -1,23 +1,41 @@
-import logo from './logo.svg';
 import './App.css';
+import FoodBox from './components/FoodBox';
+import { Row } from 'antd';
+import AddFoodForm from './components/AddFoodForm';
+import { useState } from 'react';
+import foods from "./foods.json"
+import { v4 as uuidv4 } from 'uuid'
 
 function App() {
+  foods = foods.map((food) => {
+    const id = uuidv4();
+    food.id = id;
+    return food;
+  });
+
+  const [foodList, setFoodList] = useState(foods);
+  const [filteredFoodList, setFilteredFoodList] = useState(foodList);
+
+
+  function addNewFood(newFood) {
+    const newFoodList = [...foodList, newFood]
+    setFoodList(newFoodList)
```

Comment:
here also i would set the filteredFoodList in order to have also displayed the new food that you add
```suggestion
    setFoodList(newFoodList)
        setFilteredFoodList(newFoodList)
```